moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -169,7 +169,15 @@ class TimeOptimalTrajectoryGeneration : public TimeParameterization
   bool computeTimeStamps(robot_trajectory::RobotTrajectory& trajectory, const double max_velocity_scaling_factor = 1.0,
                          const double max_acceleration_scaling_factor = 1.0) const override;
 
+  bool computeTimeStamps(robot_trajectory::RobotTrajectory& trajectory,
+                         const std::map<std::string, double>& velocity_limits,
+                         const std::map<std::string, double>& acceleration_limits) const;
+
```

Comment:
Is there a reason why the velocity and acceleration limits need to maintain their ordering, or is it worth converting these to `std::unordered_map`s?